src/Sulu/Bundle/CoreBundle/Composer/ScriptHandler.php

```diff
@@ -29,7 +29,7 @@ public static function removeComposerLockFromGitIgnore()
         }
 
         $gitignore = file_get_contents(static::GIT_IGNORE_FILE);
-        $gitignore = str_replace('composer.lock' . PHP_EOL, '', $gitignore);
+        $gitignore = str_replace("composer.lock\n", '', $gitignore);
```

Comment:
Really? But on composer-create it's still your repo and you don't make sure you only check-in `\n`?

E.g. with `git config --global core.autocrlf input`?